### ISSUE TITLE
About extracted feature

### ISSUE THREAD

**Issue by WoshiBoluo** [yangdongchao/DCASE2021Task5#1]
Hello， your work is excellent!  Could you please make public the extracted feature on Google Drive？

**Comment by yangdongchao** [comment]
> Hello， your work is excellent! Could you please make public the extracted feature on Google Drive？

I realy want to do this, but there is no space in my google drive. So I cannot share it on Google Drive now. Actually, you can use the baseline of DCASE2021 task5 to extract the feature.  I will try to share it on BaiDu cloud desk in the next few days.

**Comment by WoshiBoluo** [comment]
> > Hello， your work is excellent! Could you please make public the extracted feature on Google Drive？
> 
> I realy want to do this, but there is no space in my google drive. So I cannot share it on Google Drive now. Actually, you can use the baseline of DCASE2021 task5 to extract the feature. I will try to share it on BaiDu cloud desk in the next few days.

Thank you very much. After I used the official feature_extract, I found that the Mel_train.h5 of the train is only 152MB. Is this correct?

**Comment by yangdongchao** [comment]
> > > Hello， your work is excellent! Could you please make public the extracted feature on Google Drive？
> > 
> > 
> > I realy want to do this, but there is no space in my google drive. So I cannot share it on Google Drive now. Actually, you can use the baseline of DCASE2021 task5 to extract the feature. I will try to share it on BaiDu cloud desk in the next few days.
> 
> Thank you very much. After I used the official feature_extract, I found that the Mel_train.h5 of the train is only 152MB. Is this correct?

It is right. The training data is very small, but the validation  data is much larger.

**Comment by WoshiBoluo** [comment]
Thanks for your reply, I have another question about the evaluation set, I found that the official evaluation set provided by the .csv file only contains 5 POS labels per file, where do I get all the labels to evaluate my model?

**Comment by yangdongchao** [comment]
> Thanks for your reply, I have another question about the evaluation set, I found that the official evaluation set provided by the .csv file only contains 5 POS labels per file, where do I get all the labels to evaluate my model?

The evalution set label is only can be used during contest.

**Comment by liushenme** [comment]
Hi, your project is very interesting. I see in your paper you say the feature you used is STFT followed by a Mel-scaled filter
bank on perceptually weighted spectrograms. Is it same as PCEN baseline of the task used ?

**Comment by yangdongchao** [comment]
Yes it is the same as the baseline.

**Comment by liushenme** [comment]
> Yes it is the same as the baseline.

I see. Thank you.

**Comment by Noumanijaz744** [comment]
> > > Hello， your work is excellent! Could you please make public the extracted feature on Google Drive？
> > 
> > 
> > I realy want to do this, but there is no space in my google drive. So I cannot share it on Google Drive now. Actually, you can use the baseline of DCASE2021 task5 to extract the feature. I will try to share it on BaiDu cloud desk in the next few days.
> 
> Thank you very much. After I used the official feature_extract, I found that the Mel_train.h5 of the train is only 152MB. Is this correct?

Hello Sir @yangdongchao  the official  development datasets contains training set of  14 hours and 20 mins of audio recording while the validation set contains only 5 hours of audio recording so how the Mel_train.h5 is too smaller than the Mel_Eval.h5 .please if you can clear my question .thank you
